telegram/bot.py

```diff
@@ -144,6 +128,10 @@ def decorator(self: 'Bot', *args: Any, **kwargs: Any) -> RT:  # pylint: disable=
 class Bot(TelegramObject):
     """This object represents a Telegram Bot.
 
+    .. versionadded:: 13.2
+    Objects of this class are comparable in terms of equality. Two objects of this class are
+    considered equal, if their :attr:`bot` is equal.
+
```

Comment:
Can you build the docs locally and check if this is rendered correctly? IIRC the text below the directive needs to be indented for sphinx to see it as explanation.

Reply:
![IMG_20210116_234042_007](https://user-images.githubusercontent.com/58552395/104819681-0d16e200-5855-11eb-9388-41a84964dd1d.jpg)
Ig it looks fine 🤔

Reply:
mh, it's supposed to look like this:

![image](https://user-images.githubusercontent.com/22366557/104834808-1da17980-58a2-11eb-9eb9-2994f5e4eaef.png)

Reply:
Oops, okay